backend/routers/index_router.py

```diff
@@ -16,3 +25,17 @@ async def healthcheck():
 async def read_main():
     logger.info("In root path")
     return {"msg": "Hello World"}
+
+
+@index_router.get("/get_pixel_watch", response_model=List[schemas.PixelWatch])
```

Comment:
I don't think thats good idea to show all pixel watches. Maybe change to `get_my_pixel_watches` and return filtered list for specific user?